templates/common/on-prem/files/keepalived.yaml

```diff
@@ -54,13 +54,17 @@ contents:
         volumeMounts:
         - name: kubeconfig
           mountPath: "/etc/kubernetes"
+          mountPropagation: HostToContainer
         - name: script-dir
           mountPath: "/config"
+          mountPropagation: HostToContainer
         - name: conf-dir
           mountPath: "/etc/keepalived"
+          mountPropagation: HostToContainer
         - name: nodeip-configuration
           mountPath: "/run/nodeip-configuration"
```

Comment:
This is also a host directory. Does it need the new setting?

Reply:
I think this uses hostPath? Not sure if I am seeing something wrong

Reply:
Oh, I think the mountPropagation got added in the wrong place here. The one on line 67 should be moved up into this block.